### ISSUE TITLE
Blocking telemetry via firewall and host files

### ISSUE THREAD

**Issue by Alina23belikova** [federicadomani/TinyWinSpyOff#1]
Is it possible to add this type of blocking?

**Comment by federicadomani** [comment]
Yes, it is possible. But lists of domains and IPs to block are not static and vary from Windows version to version. Also, firewall and host file are sensitive places of Windows and I don't want to interfere them. I keep this issue open to return to its discussion in future.
